htdocs/compta/facture/card.php

```diff
@@ -528,7 +528,7 @@
 		}
 	} elseif ($action == 'setrevenuestamp' && $usercancreate) {
 		$object->fetch($id);
-		$object->revenuestamp = GETPOST('revenuestamp');
+		$object->revenuestamp = (float) GETPOST('revenuestamp', 'int');
```

Comment:
When a data for an amount or qty is coming from a user, the good way to retreive it is:
`$object->revenuestamp = (float) price2num(GETPOST('revenuestamp'));`

Reply:
OK noted.
Fixed.